build/common/init_command.sh

```diff
@@ -7,6 +7,9 @@ init_conf
 
 # Start services
 if [ "$MODULE_NAME" = 'dashboard' ]; then
+    if [ "$OPENWISP_GEOCODING_CHECK" = 'True' ]; then
```

Comment:
Why are the permissions of the file changed from `0644` to `755`?

Reply:
Maybe by mistake 😅, Could you please revert back.

Reply:
oops, by mistake :smiling_face_with_tear: